Assets/Plugins/Source/Editor/EpicOnlineServicesConfigEditor.cs

```diff
@@ -193,27 +185,22 @@ private void Awake()
             int i = 2;
             foreach (var platformSpecificConfigEditor in platformSpecificConfigEditors)
             {
-                platformSpecificConfigEditor.Awake();
-                toolbarTitleStrings[i] = platformSpecificConfigEditor.GetNameForMenu();
+                platformSpecificConfigEditor.Read();
+                toolbarTitleStrings[i] = platformSpecificConfigEditor.GetName();
                 i++;
             }
 
             LoadConfigFromDisk();
         }
 
-        private bool DoesHaveUnsavedChanges()
-        {
-            return false;
-        }
```

Comment:
In every instance where `DoesHaveUnsavedChanges` was defined, all implementations merely returned false, making it functionally meaningless, so they were all removed for the sake of clarity or until we have a clear way of implementing such a feature.

Reply:
I remember these are for saving configurations, surprises me that they weren’t used at all.

Reply:
Yes - there was some scaffolding in place that queried the return value of this function to determine whether or not to prompt the user to save settings - but again since all implementations only ever return `false`, they weren't doing anything. 

In a separate branch I actually have some work done that will re-enable this functionality in a manner that's consistent. In that branch I also implemented it so that the "Save" button on the config window will be disabled unless there are changes to save - in addition to prompting the user to save if they haven't. But it felt appropriate to restrict this PR to the scope of the problem it is intended to address.